test/gie/ellipsoid.gie

```diff
@@ -57,13 +57,8 @@ expect    failure errno unknown_ellp_param
 operation proj=merc +a=-1
 expect    failure errno major_axis_not_given
 
-operation proj=merc no_defs
-expect    failure errno major_axis_not_given
-
-# This one should succeed due to ellps=WGS84 in proj_def.dat
 operation proj=merc
-accept    0 0
-expect    0 0
+expect    failure errno major_axis_not_given
```

Comment:
Is this because we now don't have a default ellipsoid?

Reply:
yes

Reply:
I guess that would leave many projections default-less with regards to ellipsoids. It seems to me that `+proj=merc` not working without adding `+ellps=...` is a step in the wrong direction. I think setting ellipsoid parameters for `GRS80` when initialising a `PJ` would be a sane default. If `+ellps=bessel` (or whatever) is set it will obviously be overridden.